test/src/kbasesearchengine/test/events/handler/WorkspaceEventHandlerTest.java

```diff
@@ -367,14 +367,55 @@ public void loadWithNullSubActionsAndTags() throws Exception {
                 .withSourceTag("foo")
                 .withSourceTag("bar")
                 .withSourceTag("baz")
+                .withIsPublic(false)
                 .build();
         
         compare(sd, expected);
         
         verify(cloned).setStreamingModeOn(true);
         verify(cloned)._setFileForNextRpcResponse(new File("somefile"));
     }
-    
+
+    @Test
+    public void loadWithNullSubActionsAndPublic() throws Exception {
+        final CloneableWorkspaceClient clonecli = mock(CloneableWorkspaceClient.class);
+        final WorkspaceClient cloned = mock(WorkspaceClient.class);
+        final WorkspaceClient wscli = mock(WorkspaceClient.class);
+        when(clonecli.getClientClone()).thenReturn(cloned);
+        when(clonecli.getClient()).thenReturn(wscli);
+
+        when(cloned.administer(argThat(new AdminGetObjectsAnswerMatcher("2/2/3"))))
+                .thenReturn(new UObject(new GetObjects2Results().withData(Arrays.asList(
+                        new ObjectData()
+                                .withData(new UObject(ImmutableMap.of("genome", "data")))
+                                .withProvenance(Arrays.asList(new ProvenanceAction()
```

Comment:
you can simplify by just returning an empty list for the provenance and renaming the test method loadPublic() or something.

Reply:
Done